build.cake

```diff
@@ -13,6 +12,9 @@ var UNIT_TEST_ASSEMBLY = "nunit-project-loader.tests.dll";
 
 var target = Argument("target", "Default");
 var configuration = Argument("configuration", "Debug");
+var nugetVersion = Argument("nugetVersion", (string)null);
+var chocoVersion = Argument("chocoVersion", (string)null);
+var binaries = Argument("binaries", (string)null);
```

Comment:
Optional args used by the Package target. The two version args override the package version that we otherwise calculate. The `binaries` arg causes the packaging to be made from a directory other than BIN_DIR. This would have to have the desired binaries in it, from an external source. I've tested the args and they work, but I'm not yet seeing how we would get the binaries for a repackaging situation. Ideas?